src/pykx/pandas_api/pandas_meta.py

```diff
@@ -311,3 +311,18 @@ def agg(self, func, axis=0, *args, **kwargs): # noqa: C901
             return data
         else:
             return (q('{(flip enlist[`function]!enlist x)!y}', keyname, data))
+
+    @api_return
+    def isna(self):
+        tab = self
+        return q.null(tab)
+
+    @api_return
+    def isnull(self):
+        tab = self
+        return tab.isna()
+
+    @api_return
+    def notna(self):
+        tab = self
+        return q('not', tab.isna())
```

Comment:
Why not implementing `notnull` as well? Is it already done? https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.notnull.html#pandas.DataFrame.notnull